### PR TITLE
proof of concept of input controlled by convex

### LINKED ISSUE #3: Set up Convex
https://docs.convex.dev/quickstart/nextjs

For testing, send over a number, convex multiplies by 2 and sends back result to next front end

### PR DESCRIPTION
**Pull request by alcpereira**
🎉 

Closes #3 

# Description

A very simple Proof of Concept using Convex with a controlled input linked to a single row in Convex database.

# Solution

- Installed Convex package and setup a simple page
- Generated functions and schema for Convex
- Updated CI workflow to create a `.env.local` file for the build step

# Todo
Good first steps as a reviewer for TypeScript React
- [x] Find the correct type for `event` 👀 . Congrats @windyScripts for find the correct type 🥇

### CODE REVIEW

File: app/page.tsx [hunk 1]
```diff
@@ -1,113 +1,29 @@
...
-            Learn{' '}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Learn about Next.js in an interactive course with&nbsp;quizzes!
-          </p>
-        </a>
+  console.log(inputValue);
```
Comment (windyScripts) [inline app/page.tsx:10]: can we remove this console.log statement?

File: app/page.tsx [hunk 2]
```diff
@@ -1,113 +1,29 @@
...
-            Templates{' '}
-            <span className="inline-block transition-transform group-hover:translate-x-1 motion-reduce:transform-none">
-              -&gt;
-            </span>
-          </h2>
-          <p className={`m-0 max-w-[30ch] text-sm opacity-50`}>
-            Explore the Next.js 13 playground.
-          </p>
-        </a>
+  const handleOnChange = (event) => { // Feel free to investigate which would be the correct type :eyes:
```
Comment (windyScripts) [inline app/page.tsx:12]: I think this would be the ChangeEvent type.

**Review by windyScripts — DISMISSED** [review]
I really appreciate all the guiding comments. LGTM other than the non-blocking rogue console.log.

File: app/page.tsx (hunk 1 above)
Comment (alcpereira) [reply app/page.tsx]: Good catch, thanks! Feel free to open an issue with an ESLint config that will catch it ;)

File: app/page.tsx (hunk 2 above)
Comment (alcpereira) [reply app/page.tsx]: This is correct :)

**Review by HondaChan14 — APPROVED** [review]

**Pull request merged**